<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationWidget</class>
 <widget class="QWidget" name="AnimationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>animation:</string>
   </property>
  </widget>
  <widget class="AnimationViewDBChanger" name="m_pAniViewer">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m_pCurrentFrameLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>current frame:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_pCurrentFrame">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="m_pFrameTimeLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>frame time (msec):</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_pFrameTime">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pAniPlayerButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_pSpriteLabel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>sprites:</string>
   </property>
  </widget>
  <widget class="SpritePrototypeDragTreeWidget" name="m_pSpriteList">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>221</width>
     <height>261</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragOnly</enum>
   </property>
   <property name="indentation">
    <number>0</number>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="headerHidden">
    <bool>false</bool>
   </property>
   <property name="expandsOnDoubleClick">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="headerCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="headerHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="headerShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="m_pSpriteFrame">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>spriteID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>posX:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_pSpriteSpriteID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>current sprite:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_pSpritePosY">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-999999999</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>posY:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_pSpritePosX">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-999999999</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_pSpriteScale">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>scale:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>rotation:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_pSpriteRotation">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_pSpriteOpacity">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>opacity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <zorder>label_18</zorder>
  <zorder>m_pAniViewer</zorder>
  <zorder>m_pCurrentFrameLabel</zorder>
  <zorder>m_pFrameTimeLabel</zorder>
  <zorder>m_pFrameTime</zorder>
  <zorder>m_pAniPlayerButton</zorder>
  <zorder>m_pSpriteLabel</zorder>
  <zorder>m_pSpriteList</zorder>
  <zorder>m_pCurrentFrame</zorder>
  <zorder>m_pSpriteFrame</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpritePrototypeDragTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>dragtreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>AnimationViewDBChanger</class>
   <extends>QGraphicsView</extends>
   <header>animationviewwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_pAniViewer</tabstop>
  <tabstop>m_pAniPlayerButton</tabstop>
  <tabstop>m_pCurrentFrame</tabstop>
  <tabstop>m_pFrameTime</tabstop>
  <tabstop>m_pSpriteList</tabstop>
  <tabstop>m_pSpritePosX</tabstop>
  <tabstop>m_pSpritePosY</tabstop>
  <tabstop>m_pSpriteScale</tabstop>
  <tabstop>m_pSpriteOpacity</tabstop>
  <tabstop>m_pSpriteRotation</tabstop>
  <tabstop>m_pSpriteSpriteID</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
